cd/deploy.sh

```diff
@@ -1,89 +0,0 @@
-#!/usr/bin/env bash
-
-set -e
-
-repoName="Palisade"
-repoId="Palisade"
-artifactId="palisade"
-
-if [ "$TRAVIS_BRANCH" == 'master' ] && [ "$TRAVIS_PULL_REQUEST" == 'false' ]; then
-    git checkout master
-    mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version
-    POM_VERSION=`mvn org.apache.maven.plugins:maven-help-plugin:2.1.1:evaluate -Dexpression=project.version | grep -v '\['`
-    echo "POM_VERSION = $POM_VERSION"
-    if [[ "$POM_VERSION" == *SNAPSHOT ]]; then
-        if [ -z "$GITHUB_TOKEN" ]; then
-            echo "GITHUB_TOKEN has not been set. Please configure this in Travis CI settings"
-            exit 1
-        fi
-
-        if [ -z "$RELEASE_VERSION" ]; then
-            RELEASE_VERSION=${POM_VERSION%-SNAPSHOT}
-        fi
-
-        echo ""
-        echo "======================================"
-        echo "Tagging and releasing version $RELEASE_VERSION"
-        echo "======================================"
-        echo ""
-
-        # Configure GitHub token
-        git config --global credential.helper "store --file=.git/credentials"
-        echo "https://${GITHUB_TOKEN}:@github.com" > .git/credentials
-
-        # Add develop and gh-pages branches
-        git remote set-branches --add origin develop gh-pages
-        git pull
-
-        echo ""
-        echo "--------------------------------------"
-        echo "Tagging version $RELEASE_VERSION"
-        echo "--------------------------------------"
-        mvn versions:set -DnewVersion=$RELEASE_VERSION -DgenerateBackupPoms=false
-
-        git commit -a -m "prepare release $artifactId-$RELEASE_VERSION"
-        git tag $artifactId-$RELEASE_VERSION
-        git push origin --tags
-        git push
-
-        echo ""
-        echo "--------------------------------------"
-        echo "Updating doc"
-        echo "--------------------------------------"
-        npm install -g gitbook-cli > /dev/null 2>&1
```

Comment:
Might want to keep this part of the script for building the gitbook pages

Reply:
This is a script that included the building of the gitbook along with other features.  Now what we have is the scripts for gitbook as a standalone set. I don't think that we will need the deploy.sh any longer